README.md

```diff
@@ -1,46 +1,111 @@
 ## Welcome!
-The purpose of this project is to create *SCAP content* for various
-platforms -- Red Hat Enterprise Linux 6, Red Hat Enterprise Linux 7,
-Fedora, and others.  *"SCAP content"* refers to documents  in the *XCCDF*,
-*OVAL* and *Source DataStream* formats.  These documents can be presented
+
+[![Nightly ZIP Status](https://jenkins.open-scap.org/job/scap-security-guide-nightly-zip/badge/icon)](https://jenkins.open-scap.org/job/scap-security-guide-nightly-zip/) [![Nightly 5.10 ZIP Status](https://jenkins.open-scap.org/job/scap-security-guide-nightly-oval510-zip/badge/icon)](https://jenkins.open-scap.org/job/scap-security-guide-nightly-oval510-zip/) [![Link-checker Status](https://jenkins.open-scap.org/job/scap-security-guide-linkcheck/badge/icon)](https://jenkins.open-scap.org/job/scap-security-guide-linkcheck/) [![CentOS CI Status](https://ci.centos.org/job/openscap-scap-security-guide/badge/icon)](https://ci.centos.org/job/openscap-scap-security-guide/) [![landscape.io Code Health](https://landscape.io/github/OpenSCAP/scap-security-guide/master/landscape.png)](https://landscape.io/github/OpenSCAP/scap-security-guide/master)
```

Comment:
I suggest breaking badge entries one per line, it should still display them side-by-side.
And concerning the Landscape badge, I think that we should take it out as checks are not running and the displayed value is stale. We can re-add it in another PR.

Reply:
I experimented with one badge per line and IMO it didn't look good. I kept the landscape.io because it was already there, I can remove it or we can figure out wtf is going on with it in another PR. Either way, landscape was there already.

Reply:
I am OK with it, at least there is one more reason to re-add it :-)